cmake/FindFontconfig.cmake

```diff
@@ -0,0 +1,75 @@
+#[=======================================================================[.rst:
+FindFontconfig
+-------
+
+Finds the fontconfig library.
```

Comment:
Scrap that thing ;) Either just require CMake 3.14 (which has FindFontconfig), or copy that module from current CMake.